src/demo/share/jfc/SwingSet2/resources/swingset_ja.properties

```diff
@@ -48,7 +48,7 @@ SourceCode.loading=<html><body bgcolor="#ffffff">\u30BD\u30FC\u30B9\u30FB\u30B3\
 
 ### Status ###
 
-Status.loading=\u8AAD\u8FBC\u307F\u4E2D: 
+Status.loading=\u8AAD\u8FBC\u307F\u4E2D:\u0020
```

Comment:
ja and zh-CN both have spaces, but en and de don't. The string is concatenated without a space in between in the demo code, so this would be technically correct (and en and de incorrect). An example of the current behavior is producing `正在加载: 按钮演示` (correct) or `Loading:Button Demo` (incorrect).

The proper solution would be to remove the space and the colon, putting them in the code.